### DISCUSSION TITLE
Rearranging user fields

### DISCUSSION THREAD

**Discussion by chansen1029** [directus/directus#18822] [Feature Requests]
### Summary

As with any other collection that is made by the user, there is a possibility to rearrange the fields created within this collection, but for some reason, you've chosen NOT to make that possibility for the precreated collections.

I'm specifically referring to the users collection.
We've been using Directus many times and almost everytime we end up making a new field on the users which would make so much sence, to be at the top rather then having to scroll down to filled it.

I understand that there might be a reason for this, all im asking, is for the visual presentation to be able to rearrange.

### Motivation

.

### Detailed Design

.

### Requirements List

.

### Drawbacks

.

### Alternatives

.

### Adoption Strategy

.

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
